File: C#/백준/Bronze/25640. MBTI/MBTI.cs
using System;

namespace q{
    class w{
        static void Main(){
            String s = Console.ReadLine();
            int a = int.Parse(Console.ReadLine());
            int cnt = 0;
            for (int i=0; i<a; i++){
                string b = Console.ReadLine();
                if(s.Equals(b)){
                    cnt++;
                }
            }
            Console.WriteLine(cnt);
        }
    }
}

File: Java/백준/Bronze/25640. MBTI/MBTI.java
import java.util.*;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        String a = in.next();
        int b = in.nextInt();
        int cnt = 0;
        for (int i =0; i<b; i++){
            if(a.equals(in.next())){
                cnt++;
            }
        }
        System.out.println(cnt);
    }
}